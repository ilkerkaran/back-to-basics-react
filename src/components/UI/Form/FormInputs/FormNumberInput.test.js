import React from 'react';
import renderer from 'react-test-renderer';
import Number from './FormNumberInput';


describe('<Number /> snapshot ', () => {
  const inputConfig = {
    inputType: 'number',
    inputName: 'number',
    label: 'Number',
    isRequired: true,
    register: () => {},
    errors: {}
  };


  it('renders correctly', () => {
    const tree = renderer
      .create(<Number inputConfig={inputConfig}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
