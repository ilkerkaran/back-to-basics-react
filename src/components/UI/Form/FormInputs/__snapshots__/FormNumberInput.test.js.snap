// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Number /> snapshot  renders correctly 1`] = `
<div>
  <input
    className="Number"
    name="number"
    placeholder="Number"
    type="number"
  />
  <div
    style={
      Object {
        "color": "darkred",
      }
    }
  />
</div>
`;
